package.json

```diff
@@ -142,6 +142,7 @@
     "build:dev:assets": "npm-run-all build:sass:demo build:sass:component postCss:component sass:render",
     "build:markdownDocs": "node scripts/generateDocs.js",
     "build:sass": "npm-run-all build:sass:demo build:sass:component postCss:component sass:render",
+    "build:md": "npm run build:markdownDocs",
```

Comment:
This change should not be part of this PR. If we want to make a shorthand of `build:markdownDocs` (which I am in favor of) that change needs to be in WC-Generator.